packages/ui/src/components/mermaid.tsx

```diff
@@ -31,6 +34,20 @@ export const Mermaid: React.FC<{
     mermaid.contentLoaded();
   });
 
+  // requestAnimationFrame is used to ensure that the svgPanZoom is called after the svg is rendered
+  useEffect(() => {
+    requestAnimationFrame(() => {
```

Comment:
Can you walk me through this one.

Is the problem that you want to hold on the `svgPanZoom` invocation until you are sure that the diagram svg is rendered?

Reply:
Correct. If `svgPanZoom` is run before its rendered the function will error. This ensures that its run after rendering